src/nix/derivation-add.md

```diff
@@ -1,9 +1,44 @@
 R""(
 
+# Examples
```

Comment:
Examples should be after the description. Documentation looks very good other than that.

Reply:
Are you sure? I've looked through the current docs of nix3 and it seems all the commands have their examples shown first. And I like that, examples are easier to parse and often what I'm looking for when I just quickly want to jog my memory.

Reply:
Ah okay, then it's fine. I don't know all of the manual by heart. Lots of other sections have it the other way around.